docs/api/commands/fixture.mdx

```diff
@@ -183,33 +183,21 @@ cy.fixture('users').then((json) => {
 You can modify fixture data directly before visiting a URL or mounting a
 component that makes a network request to that URL.
 
-<E2EOrCtTabs>
+:::visit-mount-example
 
 ```js
 cy.fixture('user').then((user) => {
   user.firstName = 'Jane'
   cy.intercept('GET', '/users/1', user).as('getUser')
 })
 
-cy.visit('/users')
+-{cy.visit('/users')::}-cy.mount(<Users />)}-
```

Comment:
some render wonk on this one

![Screen Shot 2022-07-11 at 5 07 28 PM](https://user-images.githubusercontent.com/6298501/178358902-0b4fdae4-e075-4968-91f9-33cfea05e893.png)